frontend/src/app/plans/edit/[id]/_components/CreateNewPlanPage.tsx

```diff
@@ -252,11 +252,19 @@ export function CreateNewPlanPage({
           })
           .sort((a, b) => a.name.localeCompare(b.name));
 
-        // Aktualizacja list
+        // List update logic:
+        // Add unique registrations to the updatedRegistrations array
+        // r - current registration
+        // i - current index
+        // a - array of registrations
```

Comment:
krócej by Ci wyszło zmienić nazwy zmiennych 😭

Reply:
mozeeee, ale to jest takie ladneeeeeee

Reply:
🥺🥺🥺🥺🥺🥺🥺